index.html

```diff
@@ -117,6 +117,227 @@ <h3 class="hero-section__subtitle">I'm a software developer</h3>
         <i class="fa-solid fa-chevron-down"></i>
       </a>
     </section>
+    <main id="recent-works" class="my-main">
+      <div class="recent-works__title--wrapper">
+        <h1 class="recent-works__title">My Recent Works</h1>
+        <hr class="recent-works__hr-line" />
+      </div>
+      <section class="recent-works projects-container container">
+        <div class="projects-card">
+          <img
+            src="src/assets/media/img-placeholder.png"
+            alt="template will be replaced by a project soon"
+            class="projects-card__img"
+          />
+          <div class="projects-card__title--wrapper">
+            <p class="projects-card__title">Multi-Post Stories</p>
```

Comment:
-kindly, use header tag `hx` for titles and apply this for all card titles for semantic reasons 👌

Reply:
Hi @binhussen! thank you for your review! 
Yes, I had my doubts about the -p- tags for this specific section. Will fix it right away!🎯